### PR TITLE
[REVIEW] Update Azure Notebook to use CloudML container

### PR DESCRIPTION
**Pull request by Nanthini10**
See https://github.com/rapidsai/cloud-ml-examples/issues/58 

Replace conda environment setup with CloudML docker container.

### CODE REVIEW

File: azure/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ We recommend using RAPIDS docker image on your local system and using the same i
 
 In the example notebook we are using `rapidsai/rapidsai-core:cuda10.2-runtime-ubuntu18.04-py3.8` docker image, to pull and run this use the following command. The `-v` flag sets the volume you'd like to mount on the docker container. This way, the changes you make within the docker container are present on your local system to. Make sure to change `local/path` to the path which contains this repository.
 
-    `docker run --runtime nvidia --rm -it -p 8888:8888 -p 8787:8787 -v /local/path:/docker/path rapidsai/rapidsai-core:cuda10.2-runtime-ubuntu18.04-py3.8`
+`docker run --runtime nvidia --rm -it -p 8888:8888 -p 8787:8787 -v /local/path:/docker/path rapidsai/rapidsai-core:cuda10.2-runtime-ubuntu18.04-py3.8`
```
Comment (zronaghi) [inline azure/README.md:69]: Would recommend updating to cuda 11

File: azure/notebooks/HPO-RAPIDS.ipynb [hunk 2]
```diff
@@ -337,29 +337,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from azureml.core import Environment\n",
+    "from azureml.core.container_registry import ContainerRegistry\n",
     "\n",
-    "# create the environment\n",
-    "rapids_env = Environment('rapids_env')\n",
+    "#this is an image available in Docker Hub\n",
```
Comment (zronaghi) [inline azure/notebooks/HPO-RAPIDS.ipynb:342]:
```suggestion
    "#rapids-cloud-ml image is available in Docker Hub\n",
```

File: azure/notebooks/HPO-RAPIDS.ipynb [hunk 3]
```diff
@@ -337,29 +337,13 @@
...
-    "pip install azureml-mlflow\n",
-    "\"\"\"\n",
-    "\n",
-    "# set base image to None since the image is defined by dockerfile\n",
-    "rapids_env.docker.base_image = None\n",
-    "rapids_env.docker.base_dockerfile = dockerfile\n",
-    "\n",
-    "# use rapids environment in the container\n",
-    "rapids_env.python.user_managed_dependencies = True"
+    "#don't let the system build a new conda environment\n",
```
Comment (zronaghi) [inline azure/notebooks/HPO-RAPIDS.ipynb:345]:
```suggestion
    "#use rapids environment in the container, don't build a new conda environment\n",
```

**Review by zronaghi — CHANGES_REQUESTED** [review]
Looks good, just a few small comments and will merge after updating Train-RAPIDS notebook as well

File: azure/README.md (hunk 1 above)
Comment (Nanthini10) [reply azure/README.md:69]: I'll update this in the PR for Azure MNMG notebook, since the notebook has to be changed as well.

**Review by zronaghi — APPROVED** [review]

**Pull request merged**